bindings/stronghold-nodejs/js/stronghold.ts

```diff
@@ -97,14 +97,14 @@ export class Stronghold implements Storage {
         return Signature.fromJSON(napiSignature.toJSON());
     }
 
-    public async chainStateGet(did: DID): Promise<ChainState | undefined | null> {
+    public async chainStateGet(did: DID): Promise<ChainState | undefined> {
         const napiDID: NapiDID = NapiDID.fromJSON(did.toJSON());
-        const napiChainState: NapiChainState | undefined | null = await this.napiStronghold.chainStateGet(napiDID);
+        const napiChainState: NapiChainState | undefined = await this.napiStronghold.chainStateGet(napiDID);
```

Comment:
I believe the `undefined | null` thing in parameters and return values is due to Napi `Options` always generating Typescript with `undefined | null`, not sure if it matters. I would say this should just match whatever the `Storage` interface from Wasm uses.

Same comment for `documentGet`.

Reply:
I changed it in `WasmStorage` to `ChainState | undefined` for `chainStateGet`, similarly for `documentGet`, so this should be okay as it is now.